gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/MinioConfiguration.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.gridcapa_cse.app.configurations;
+
+import com.farao_community.farao.gridcapa_cse.api.exception.CseInternalException;
+import io.minio.MinioClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author Amira Kahya {@literal <amira.kahya at rte-france.com>}
+ */
+@Configuration
+public class MinioConfiguration {
+
+    @Value("${minio.access.name}")
+    private String accessKey;
+    @Value("${minio.access.secret}")
+    private String accessSecret;
+    @Value("${minio.url}")
+    private String minioUrl;
+    @Value("${minio.bucket}")
+    private String bucket;
+    @Value("${minio.base-path}")
+    private String basePath;
```

Comment:
cse-cc:
  -- minio:
    ---- connect:
      ------ url: ...
      ------ access-key: ...
      ------ secret-key: ...
    ---- bucket: ...
    ---- base-directory: ...

I feel like something like that would be more inline with what we did in other services